src/lib/cards/MissionCompletionCard.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+	import type { IndividualCompletion } from '$lib/types';
+	import { getPersonColor, listify } from '$lib/util';
+
+	export let mission: IndividualCompletion;
+	export let username: string;
+	const methods = [
+		mission.defuser ? 'as a Defuser' : null,
+		mission.expert ? 'as an Expert' : null,
+		mission.efm ? 'via EFM' : null,
+		mission.solo ? 'solo' : null
+	].flatMap(method => method ?? []);
+	const title = `${username} solved this mission ${listify(methods)}.`;
+</script>
+
+<a class="mission" href="/mission/{encodeURIComponent(mission.name)}">
+	<div class="mission-name" class:green={mission.defuser && mission.expert && mission.efm}>
+		{mission.name}
+	</div>
+	<div {title}>
+		<div
+			class:hidden={!mission.defuser}
```

Comment:
For something that is going to be displayed many times, it would be better to use an `{#if ...}` block. This will reduce the number of elements the browser has to display, making the page faster.